components/Searchbar/Searchbar.js

```diff
@@ -0,0 +1,33 @@
+import styled from "styled-components";
+
+const SearchBarContainer = styled.form`
+  display: flex;
+  justify-content: center;
+  margin-bottom: 0.5rem;
+`;
+
+const SearchInput = styled.input`
+  padding: 0.3rem;
+  font-size: 0.9rem;
+`;
+
+const SearchButton = styled.button`
+  font-size: 1rem;
+`;
+
+export default function SearchBar({ setSearchTerm }) {
+  return (
+    <>
```

Comment:
We don't need the React Fragment in this case as we are only returning one thing from the function `SearchBarContainer`.

Reply:
Thanks, fixed!